src/azul/openapi/validation.py

```diff
@@ -47,8 +47,10 @@ def __init__(self,
         if self.schema:
             parameter_schema = self.schema
         else:
+            # noinspection PyUnresolvedReferences
             for k, v in self.content.items():
                 parameter_schema = v['schema']
+        # noinspection PyUnboundLocalVariable
```

Comment:
I'm debating if i should remove this check, as the openapi schema validation tests would throw errors if the API schema is defined in a way where this would raise a type error.